arm/Microsoft.OperationalInsights/workspaces/linkedServices/deploy.bicep

```diff
@@ -0,0 +1,41 @@
+@description('Required. Name of the Log Analytics workspace')
+param logAnalyticsWorkspaceName string
+
+@description('Required. Name of the link')
+param name string
+
+@description('Required. The resource id of the resource that will be linked to the workspace. This should be used for linking resources which require read access.')
+param resourceId string = ''
+
+@allowed([
+  'Deleting'
+  'ProvisioningAccount'
+  'Succeeded'
+  'Updating'
```

Comment:
Is this an actual property we can interact with?

Reply:
This is one of the properties listed in the documentation hete: https://docs.microsoft.com/en-us/azure/templates/microsoft.operationalinsights/workspaces/linkedservices?tabs=bicep#linkedserviceproperties It sounded weird to me as well. Let me try and remove that parameter for now and see if still works.

Reply:
removed this parameters, @eriqua and @ahmadabdalla